R/dataset.R

```diff
@@ -200,6 +200,39 @@ Dataset.query <- function(id, paginate=FALSE, env = solvebio:::.solveEnv, ...) {
 }
 
 
+#' Dataset.fields
+#'
+#' Retrieves the list of fields and field metadata for a dataset.
+#'
+#' @param id The ID of a SolveBio dataset, or a Dataset object.
+#' @param env (optional) Custom client environment.
+#' @param ... (optional) Additional query parameters (e.g. limit, offset).
+#'
+#' @examples \dontrun{
+#' Dataset.fields("1234567890")
+#' }
+#'
+#' @references
+#' \url{https://docs.solvebio.com/}
+#'
+#' @export
+Dataset.fields <- function(id, env = solvebio:::.solveEnv, ...) {
+    if (class(id) == "numeric") {
+        warning("Please use string IDs instead of numeric IDs.")
+    }
```

Comment:
should we add this to all Solve ID lookups?

Reply:
Yes